changelogs/.gitignore

```diff
@@ -0,0 +1 @@
+/.plugin-cache.yaml
```

Comment:
Do you think this is required since we have already [here](https://github.com/ansible-collections/ansible.posix/blob/de75c6f3252a196007dac6c28e7a81c99382ae45/.gitignore#L388)?

Reply:
@Akasurde no, I actually don't know how that got in there ... must have messed up.

Reply:
fixed!